tempesta_fw/t/unit/test_http_sticky.c

```diff
@@ -377,7 +377,8 @@ http_sticky_suite_teardown(void)
 			mock.req->sess->st_conn.srv_conn = NULL;
 		tfw_http_msg_free((TfwHttpMsg *)mock.req);
 	}
-	tfw_http_msg_free((TfwHttpMsg *)mock.resp);
+	if (mock.resp)
```

Comment:
The change is not really required since `tfw_http_msg_free` can handle NULL pointers inside it. but it's OK to have the condition here.

Reply:
Yeah, just wanted to have the code for requests and response freeing consistent.